src/libsyntax/feature_gate.rs

```diff
@@ -1102,11 +1103,18 @@ pub const BUILTIN_ATTRIBUTES: &'static [(&'static str, AttributeType, AttributeG
                                    "infer 'static lifetime requirements",
                                    cfg_fn!(infer_static_outlives_requirements))),
 
+    // RFC 2070 (deprecated attribute name)
+    ("panic_implementation",
+     Normal, Gated(Stability::Deprecated("https://github.com/rust-lang/rust/issues/44489#issuecomment-415140224"),
```

Comment:
Just to double check: `Stability::Deprecated` implies unstable, right?

Reply:
You still need `#![feature(panic_implementation)]` to use the deprecated `#[panic_implementation]` attribute (see recently added ui / feature-gate test) so it remains a nightly-only feature.